lib/bbc-microbit-io.js

```diff
@@ -137,21 +137,27 @@ BBCMicrobitIO.prototype.digitalRead = function(pin, handler) {
 BBCMicrobitIO.prototype._onDiscover = function(microbit) {
   debug('on discover', microbit);
 
-  this._microbit = microbit;
+  //Certain Intel BT Chipsets seem to require a scan stop before connections are allowed, otherwise they'll throw a: Connection Rejected due to Limited Resources 
+  BBCMicrobit.stopScanning(() => {
```

Comment:
What's the minimum version of Node.js needed for the `() => {` syntax? I'm leaning towards just keeping it as an old school function here. Thoughts?